<p>All language keywords must be fully spelled out.</p>

<h2>Noncompliant Code Example</h2>
<pre>
 DEF VAR xx AS CHAR NO-UNDO.
</pre>

<h2>Compliant Solution</h2>
<pre>
 DEFINE VARIABLE xx AS CHARACTER NO-UNDO.
</pre>
