<p>In your BUFFER-COPY statement you listed a field that only exists in the target table. The list of fields in the EXCEPT
or USING phrase should be in the terms of the source table.</p>

<h2>Noncompliant Code Example</h2>
<pre>

</pre>

<h2>Compliant Solution</h2>
<pre>

</pre>
