<p>All buffer references (including database tables, temp-tables, and buffers) must be fully qualified.</p>

<h2>Noncompliant Code Example</h2>
<pre>
 /* On sports2000 database */
 FIND FIRST Customer.
 DISPLAY CustNum.
</pre>

<h2>Compliant Solution</h2>
<pre>
 FIND FIRST Customer.
 DISPLAY Customer.CustNum.
</pre>
