<p>An error occured during the code parsing phase. In order to improve the overall quality of the parser, please report those errors on <a href="https://github.com/Riverside-Software/sonar-openedge">GitHub</a>.</p>
<p>Some parsing errors won't be fixed due to high development cost ; in this case, it should be possible to slightly modify the syntax so that the parser will accept it.</p>

<h2>Noncompliant Code Example</h2>
<pre>

</pre>

<h2>Compliant Solution</h2>
<pre>

</pre>
