<h2>Why is this an issue?</h2>
<p>This issue is raised whenever the OpenEdge compiler finds code which will never be executed.</p>

<h2>How to fix it</h2>
<h3>Noncompliant Code Example</h3>
<pre>
PROCEDURE Test:
  MESSAGE "Test".
  RETURN.
  MESSAGE "Test2". /* RETURN statement makes this MESSAGE unreachable */
END PROCEDURE.
</pre>

<h3>Compliant Solution</h3>
<pre>
PROCEDURE Test:
  MESSAGE "Test".
  MESSAGE "Test2".
  RETURN.
END PROCEDURE.
</pre>
