<h2>Why is this an issue?</h2>
<p>The statement was analyzed by the compiler as an expression statement, but the expression compiled to a constant, which will be a no-op at run-time. You might want to check to see if the previous statement in your program was prematurely terminated by a period or a colon by mistake.</p>

<h2>How to fix it</h2>
<h3>Noncompliant Code Example</h3>
<pre>
ASSIGN 1 = 0. // Can be removed
</pre>
