<h2>Why is this an issue?</h2>
<p>This issue is raised whenever a TRANSACTION keyword is found within the actual scope of a transaction, meaning that the TRANSACTION keyword doesn't have any effect.</p>

<h2>How to fix it</h2>
<h3>Noncompliant Code Example</h3>
<pre>
FIND FIRST Customer EXCLUSIVE-LOCK.
DO TRANSACTION:
  FIND FIRST Item.
  ASSIGN Item.Name = "xx".
END.
</pre>

<h3>Compliant Solution</h3>
<pre>
/* There are different compliant solutions, one may be : */
DO TRANSACTION:
  FIND FIRST Customer EXCLUSIVE-LOCK.
  ASSIGN Customer.Name = "xx".
  FIND FIRST Item.
  ASSIGN Item.Name = "xx".
END.
</pre>
