<h2>Why is this an issue?</h2>
<p>All language keywords must be fully spelled out.</p>

<h2>How to fix it</h2>
<h3>Noncompliant Code Example</h3>
<pre>
DEF VAR xx AS CHAR NO-UNDO.
</pre>

<h3>Compliant Solution</h3>
<pre>
DEFINE VARIABLE xx AS CHARACTER NO-UNDO.
</pre>
